observability/arobit/deploy/templates/forwarder-configmap.yaml

```diff
@@ -1,3 +1,5 @@
+{{- $kustoRegion := get ( .Files.Get "kusto-regions.yaml" | fromYaml ) .Values.forwarder.geoShortId -}}
```

Comment:
Why do we need to do the reverse lookup? Can't we use the geoShortID as the DNS segment in the Kusto link? I guess it's fine to have this way, but remembering to keep the `kusto-regions.yaml` always up-to-date with every possible cloud & region sounds like a bad idea. If we can remove the need for the reverse lookup, that would be better